homeassistant/components/elkm1/config_flow.py

```diff
@@ -357,7 +363,7 @@ async def async_step_import(self, user_input):
             return self.async_abort(reason=list(errors.values())[0])
         return result
```

Comment:
Wasn't sure of the "right" fix here, without reworking `_async_create_or_error`. Typing wants me to check not None on each of the two return values. i.e.: to satisfy typing the code would have to look something like:
```
if not error:
    return error
if not result:
    return result
return ??? (must be a FlowResult)
```

Reply:
To help mypy understand these cases, you can add an assert statement:
```python
    if result is not None:
        return result
    assert errors is not None
    return self.async_abort(reason=list(errors.values())[0])
```